src/main/java/org/tron/common/runtime/RuntimeImpl.java

```diff
@@ -141,7 +137,7 @@ public RuntimeImpl(Transaction tx, BlockCapsule block, DepositImpl deposit,
     this.isStaticCall = isStaticCall;
   }
 
-  public RuntimeImpl(Transaction tx, BlockCapsule block, DepositImpl deposit,
+  private RuntimeImpl(Transaction tx, BlockCapsule block, DepositImpl deposit,
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused private "RuntimeImpl" constructor. ([squid:UnusedPrivateMethod](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AUnusedPrivateMethod))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1681&issues=AWbnEHMySgN4uCcljsrr&open=AWbnEHMySgN4uCcljsrr)

[ISSUE_KEY]: AWbnEHMySgN4uCcljsrr